manager/src/main/java/com/redhat/service/smartevents/manager/models/ServiceLimitInstance.java

```diff
@@ -0,0 +1,50 @@
+package com.redhat.service.smartevents.manager.models;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class ServiceLimitInstance {
```

Comment:
I personally think this would be better named as `ServiceLimit` or `InstanceLimit`.

Reply:
done